code/renderers/server/src/config.ts

```diff
@@ -1,3 +1,4 @@
 export { render, renderToCanvas } from './render';
 
 export const parameters = { renderer: 'server' as const };
+export default {};
```

Comment:
@JReinhold — Any idea how/why this change is part of this PR?

Reply:
Rebasing onto the latest `next` removed it. 🤷‍♂️

Reply:
My bad, I'm pretty sure I was just testing stuff out, shouldn't be there .